### ISSUE TITLE
learning rate may be wrong 

### ISSUE THREAD

**Issue by priyankclarifai** [wangguanan/Pytorch-Person-REID-Baseline-Bag-of-Tricks#3]
while I was checking the learning rate, some discrepancies were noticed from the original paper

`def get_lr(self):
		current_epoch = self.last_epoch + 1

		if current_epoch <= self.warmup_iters  :

			return [ base_lr / 10 * float(self.last_epoch + 1 ) / float(self.warmup_iters) for base_lr in self.base_lrs ]

		elif current_epoch <= 40:

			return [base_lr  for base_lr in self.base_lrs ]

		elif current_epoch <= 70:

			return [base_lr / 10 for base_lr in self.base_lrs ]

		elif current_epoch <= 120:

			return [base_lr / 100  for base_lr in self.base_lrs ]
`

**Comment by wangguanan** [comment]
Hi, we also use warm-up when less than 10 epochs and learning-rate-decay with milestones [40, 70]. The settings are the same as the original paper's. 
Please see core/base.py/WarmupMultiStepLR for details.

**Comment by priyankclarifai** [comment]
I was getting a different values for the incremental warmup step for epochs less than <10 epochs.
